website/docs/r/elastic_cloud_elasticsearch.html.markdown

```diff
@@ -39,7 +39,10 @@ The following arguments are supported:
 
 * `resource_group_name` - (Required) The name of the Resource Group where the Elasticsearch resource should exist. Changing this forces a new Elasticsearch to be created.
 
-* `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.
+* `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.\
+Please note that the SKU depends on the Elasticsearch Plans available for your account and is a combination of PlanID_Term.\
```

Comment:
Can we keep the convention for NOTE:

```
-> **NOTE:** <The actual content...>
```

Reply:
Done in https://github.com/hashicorp/terraform-provider-azurerm/pull/23863/commits/aca5a7a845c9bc500af12eb9c6e9ba38f8f5ca59